explorer_frontend/src/styleHelpers.ts

```diff
@@ -12,20 +12,37 @@ export const getMobileStyles = (styles: StyleObject) => ({
 
 export const getTabletStyles = (styles: StyleObject) => ({
   [`@media screen and (min-width: ${mobileMaxScreenSize + 1}px) and (max-width: ${tabletMaxScreenSize}px)`]:
-    {
-      ...styles,
-    },
+  {
+    ...styles,
+  },
+});
+
+export const getDesktopStyles = (styles: StyleObject) => ({
+  [`@media screen and (min-width: ${tabletMaxScreenSize + 1}px)`]: {
+    ...styles,
+  },
 });
 
 export const getMediumScreenStyles = (styles: StyleObject) => ({
   [`@media screen and (min-width: ${tabletMaxScreenSize + 1}px) and (max-width: ${mediumMaxScreenSize}px)`]:
-    {
-      ...styles,
-    },
+  {
+    ...styles,
+  },
 });
 
 export const getLargeScreenStyles = (styles: StyleObject) => ({
   [`@media screen and (min-width: ${mediumMaxScreenSize + 1}px)`]: {
     ...styles,
   },
 });
+
+export const tableContainerStyles: StyleObject = {
+  overflowX: "auto" as const,
```

Comment:
you can reuse `getMobileStyles` etc here instead of writing raw queries btw in order to make queries setup only once.